xbmc/filesystem/test/TestFile.cpp

```diff
@@ -52,11 +52,11 @@ TEST(TestFile, Read)
   file.Flush();
   currentPos = firstBuf.length();
   EXPECT_EQ(currentPos, file.GetPosition());
-  EXPECT_TRUE(memcmp(firstBuf.c_str(), buf, firstBuf.length()) == 0);
-  EXPECT_TRUE(file.Read(buf, secondBuf.length()));
+  EXPECT_EQ(0, memcmp(firstBuf.c_str(), buf, firstBuf.length()));
+  EXPECT_EQ(secondBuf.length(), file.Read(buf, secondBuf.length()));
```

Comment:
this one changes test behavior (before we tested if we have read anything - now we test if we have read the whole length) - might not be an issue but just pointing it out ...